Task5.3.2

```diff
@@ -0,0 +1,24 @@
+def check_perfect(n):
+    """Takes number and checks if the number is perfect.
```

Comment:
What is a perfect number?
What is n? 
Please describe more, and dont use single letter names (change n).